docs/user-import.md

```diff
@@ -0,0 +1,67 @@
+# Importing users into identity
+
+## Overview
+
+As more and more DfE services migrate to using `Identity` to manage authentication and authorisation, there is the need to migrate existing users of these services into identity.  
+While this could be done the next time the user interacts with the specific service, it would mean them having to re-enter their details in order to create a teaching account.  
+The user import feature has been created to automate the creation of teaching accounts in identity from users of other DfE services as much as possible.
+
+## Import File Definition
+
+The user import requires a CSV file with a header with the following fields:
+
+| Header Name   | Description                                                  | Expected Format                                    |
+| ------------- | ------------------------------------------------------------ | -------------------------------------------------- |
+| ID            | The unique ID associated with the user in the source service | A string up to 100 characters                      |
+| EMAIL_ADDRESS | The user's email address                                     | A valid email address format up to 200 characters* |
+| FIRST_NAME    | The user's first name                                        | A string up to 200 characters                      |
+| LAST_NAME     | The user's last name                                         | A string up to 200 characters                      |
+| DATE_OF_BIRTH | The user's date of birth                                     | A valid date in ddMMyyyy format e.g. 03051971      |
+| TRN           | The user's TRN (if known in the source service)              | Empty or a 7 digit number                          |
+
+\* Note that there is no validation of whether the email address supplied is actually a valid personal email
+
+## Download File Definition
+
+The results of each file import can be downloaded in a CSV file with the following fields:
+
+| Header Name         | Description                                                  | Format                                         |
```

Comment:
Now I'm seeing it in doc form, I wonder whether we should change the header format out this output file. Seems a little strange to have ONE_CASING for the import and AnotherCasing for the output.

Reply:
Could change the output one maybe ?

Reply:
Funny I sort of thought the same

Reply:
I've fixed this and updated the docs